### PR TITLE
Add debug

### PR DESCRIPTION
**Pull request by soulgalore**
@gmierz I think this is ok now from my side, do you think it will work for you?

See also #1790

### CODE REVIEW

**Review by gmierz — CHANGES_REQUESTED** [review]
This looks great @soulgalore! :)

I tested it out locally and the only thing that's missing here is to not close the browser in debug mode. Otherwise, the browser gets closed even if it's in detached mode.

File: lib/support/cli.js [hunk 1]
```diff
@@ -1159,6 +1167,12 @@ module.exports.parseCommandLine = function parseCommandLine() {
       describe:
         'The wait time before you start the real testing after your pre-cache request.'
     })
+    .option('debug', {
+      type: 'boolean',
+      default: false,
+      describe: 'Run the Browsertime in debug mode.',
```
[the remark this answers is not in the document]
Comment (soulgalore) [reply lib/support/cli.js:1173]: You are right, I don't know where I got that from.

**Comment by soulgalore** [comment]
> I tested it out locally and the only thing that's missing here is to not close the browser in debug mode. Otherwise, the browser gets closed even if it's in detached mode.

Ah I missed that. I'll add that for Firefox!

**Comment by gmierz** [comment]
Thank you so much for taking care of this @soulgalore! :)

**Pull request merged**